### PR TITLE
tikv: change default store/apply max-batch-size

### PR DESCRIPTION
**Pull request by gengliqi**
Signed-off-by: gengliqi <gengliqiii@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs/pull/5168
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@gengliqi could you invite a technical review?

**Review by BusyJay — APPROVED** [review]

**Review by NingLin-P — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -593,7 +593,7 @@ raftstore 相关的配置项。
 ### `store-max-batch-size`
 
 + 一轮处理的最大请求个数。
-+ 默认值：1024
++ 默认值：如果开启 `hibernate-regions` 为 256，否则为 1024
```
Comment (TomShawn) [inline tikv-configuration-file.md:596]:
```suggestion
+ 默认值：如果开启 `hibernate-regions`，默认值为 256；如果关闭 `hibernate-regions`，默认值为 1024
```

**Review by yikeke — APPROVED** [review]

**Pull request merged**